winpr/libwinpr/utils/json/json.c

```diff
@@ -617,7 +617,14 @@ BOOL WINPR_JSON_AddItemToArray(WINPR_JSON* array, WINPR_JSON* item)
 		return FALSE;
 	return TRUE;
 #elif defined(WITH_CJSON)
+#if defined(USE_CJSON_COMPAT)
+	if ((array == NULL) || (item == NULL))
```

Comment:
maybe a good idea to check for a specific version?
`CJSON_VERSION_MAJOR` et al, so this is only used when compiled against a too old version?